### DISCUSSION TITLE
Official OKD 4.16 release date?

### DISCUSSION THREAD

**Discussion by rasystems1** [okd-project/okd#2045] [Q&A] [answered]
Hello!, I have been finding out about the OKD project and its modification from FCOS to SCOS, but many points are not clear to me, regarding its upcoming launch date, since I have found different questions and statements indicating that in recent weeks it was leaving to launch an official document with several points to follow, what status is this in? 

Also, I have tried to install the stable version of OKD 4.16 SCOS in vSphere mode and it fails when booting with Secure boot, is there any possible solution to boot it with secure boot activated? Can it be used in production this 4.16 stable version? Or when the GM is launched will it appear renamed with that name? I have several doubts because there are several people installing and updating but I see that there is no GM version yet, and I don't know if it is coming soon or its vision at the end of the year. Thank you very much in advance, greetings!

**Comment by nate-duke** [comment]
https://github.com/okd-project/okd/discussions/2019#discussioncomment-10982753

This is the closest thing we have to an ETA/Status and was just posted late last week.

**Comment by jcpowermac** [comment]
The backport of secureboot disable for 4.16 has been merged two weeks ago
https://github.com/openshift/installer/pull/8904

Just extract the updated installer from the release image.
https://amd64.origin.releases.ci.openshift.org/releasestream/4.16.0-0.okd-scos/release/4.16.0-0.okd-scos-2024-10-22-014547

**Comment by JaimeMagiera** [comment]
We do not currently have a final 4.16 release. Several nightly builds were dropped into the stable channel a few weeks ago because they passed AWS and vSphere IPI end-to-end tests (which is how OKD used to be released). We need more tasting from the community to find issues outside of IPI installs on the above platforms. As mentioned above, the SecureBoot issue has been resolved. We’re hoping to officially release 4.16 in November.

**Comment by Akitash** [comment]
Is there any update on a release date for OKD? I've seen that you've published versions with prefixes like 0 and 1.

**Reply by GingerGeek** [reply]
We are in the process of releasing and getting all the announcements together etc. 4.16 will be a pass-thru only release and we will expect people to proceed straight to 4.17

**Comment by GingerGeek** [comment, marked as answer]
We have now "officially" released both 4.16 and 4.17. Please do report any issues you come across.

https://okd.io/blog/2024/12/16/okd-4-16-and-4-17-release/

**Comment by stanbog** [comment]
Thank you for all the hard work in getting the 4.16 & 4.17 released.

Quick question:
Is there a way to do the upgrade from the GUI ? When I select the '4-scos-stable' channel I get the following:

<img width="853" alt="Screenshot 2024-12-18 at 18 57 06" src="https://github.com/user-attachments/assets/cf0bb295-71a7-47f4-8281-f66bf0339a6a" />

**Comment by oskarion** [comment]
![image](https://github.com/user-attachments/assets/ac782f5d-2d45-4e30-83fe-d94349955200)

Will be there information in GUI to update to 4.16 and 4.17? Is there a need to change update channel or 4.16 is not tagged as stable for now?

**Reply by 1b0gu** [reply]
Have you read https://okd.io/blog/2024/12/16/okd-4-16-and-4-17-release/ ?
It's all there except explanation how to handle issue with orphan_file parameter in /boot filesystem. I workaround it by disabling the feature with tune2fs on each control plane and worker cluster node by booting each of machines from ISO image.

Simply check the output of:
```
tune2fs -l /dev/sda3 |grep features
```
If you don't see orphan_file on the list then you're good to go with the upgrade procedure described there
